src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -151,6 +151,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $tag = $this->findProperTagName($input, $errorIo, $object, $tag);
             $options = ['tag' => $tag];
         } elseif ($name = $input->getArgument('name')) {
+            if (!$input->getOption('show-arguments')) {
+                $errorIo->warning('Not passing the --show-arguments option is deprecated when a service name is provided.');
```

Comment:
If we add a `--hide-arguments` option, this is what the message might look like.
```
Arguments will be show by default when a service name is provided in Symfony 8.0. Pass one of the options --show-arguments or --hide-arguments to hide this warning.
```